src/wallet/wallet.h

```diff
@@ -791,6 +791,21 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     bool FundTransaction(CMutableTransaction& tx, CAmount& nFeeRet, bool overrideEstimatedFeeRate, const CFeeRate& specificFeeRate, int& nChangePosInOut, std::string& strFailReason, bool includeWatching, bool lockUnspents, const std::set<int>& setSubtractFeeFromOutputs, bool keepReserveKey = true, const CTxDestination& destChange = CNoDestination());
 
     /**
+     * Bumps the fee of an opt-in-RBF transaction <txid>,
+     * replacing it with a new transaction <wtxNew>
+     */
+    enum BumpFeeResult
+    {
+        BumpFeeResult_OK,
+        BumpFeeResult_INVALID_ADDRESS_OR_KEY,
+        BumpFeeResult_INVALID_REQUEST,
+        BumpFeeResult_INVALID_PARAMETER,
+        BumpFeeResult_WALLET_ERROR,
+        BumpFeeResult_MISC_ERROR,
+    };
```

Comment:
The function doesn't seem to return anything and the enum is not used anywhere it seems.
Perhaps the function could start as void in this code movement and change it afterwards?